pkg/specgen/generate/kube/kube.go

```diff
@@ -170,7 +172,12 @@ func ToSpecGen(ctx context.Context, opts *CtrSpecGenOptions) (*specgen.SpecGener
 			}
 			s.LogConfiguration.Size = logSize
 		default:
-			s.LogConfiguration.Options[split[0]] = split[1]
+			// tags only for journald
+			if len(split[1]) > 0 && (s.LogConfiguration.Driver == "" || s.LogConfiguration.Driver == "jorunald") {
```

Comment:
Nit, non-blocker: I think if `len(split[1]) <= 0` it should give `invalid log option` or something similar as of now it will give `logrus.Warn("can only set tags with journald driver")` which might be misleading for scenario when `len(split[1]) <= 0`